cupy/cuda/memory.pyx

```diff
@@ -1154,6 +1231,46 @@ cdef class MemoryPool(object):
         mp = <SingleDeviceMemoryPool>self._pools[device.get_device_id()]
         return mp.total_bytes()
 
+    cpdef set_limit(self, size=None, fraction=None):
+        """Set the upper limit of memory allocation of the current device.
```

Comment:
"Sets"

Reply:
Thanks, fixed methods of other docs in MemoryPool too.